src/test/scala/com/redislabs/provider/redis/sql/SparkSqlStandaloneSuite.scala

```diff
@@ -0,0 +1,110 @@
+    val tableName = Person.generatePersonTableName()
+    // TODO: retrieve table name from tableName instead of path option
+    spark.sql(
+      s"""CREATE TEMPORARY TABLE $tableName (name STRING, age INT, address STRING, salary DOUBLE)
+         |  USING $RedisFormat OPTIONS (path '$tableName')
+         |""".stripMargin)
+    spark.sql(
+      s"""INSERT INTO TABLE $tableName
+         |  VALUES ('John', 30, '60 Wall Street', 150.5),
+         |    ('Peter', 35, '110 Wall Street', 200.3)
+         |""".stripMargin)
+    val loadedDf = spark.read.format(RedisFormat).load(tableName).cache()
+    verifyDf(loadedDf)
+  }
+
+  test("create temporary table then make overwrite insertions when no data exists") {
+    val tableName = Person.generatePersonTableName()
+    // TODO: retrieve table name from tableName instead of path option
+    spark.sql(
+      s"""CREATE TEMPORARY TABLE $tableName (name STRING, age INT, address STRING, salary DOUBLE)
+         |  USING $RedisFormat OPTIONS (path '$tableName')
+         |""".stripMargin)
+    spark.sql(
+      s"""INSERT overwrite TABLE $tableName
+         |SELECT * FROM VALUES ('John', 30, '60 Wall Street', 150.5),
+         |    ('Peter', 35, '110 Wall Street', 200.3)
+         |""".stripMargin)
+    val loadedDf = spark.read.format(RedisFormat).load(tableName).cache()
+    verifyDf(loadedDf)
+  }
+
+  test("create temporary table then make overwrite insertions when data exists") {
+    val tableName = Person.generatePersonTableName()
+    // TODO: retrieve table name from tableName instead of path option
+    spark.sql(
+      s"""CREATE TEMPORARY TABLE $tableName (name STRING, age INT, address STRING, salary DOUBLE)
+         |  USING $RedisFormat OPTIONS (path '$tableName')
+         |""".stripMargin)
+    spark.sql(
+      s"""INSERT INTO TABLE $tableName
+         |  VALUES ('Johnny', 18, '17 Home Street', 0),
+         |    ('Peter', 23, '6 Home Street', 20)
+         |""".stripMargin)
+    spark.sql(
+      s"""INSERT overwrite TABLE $tableName
+         |SELECT * FROM VALUES ('John', 30, '60 Wall Street', 150.5),
+         |    ('Peter', 35, '110 Wall Street', 200.3)
+         |""".stripMargin)
+    val loadedDf = spark.read.format(RedisFormat).load(tableName).cache()
+    verifyDf(loadedDf)
+  }
+
+  test("create temporary table, make regular insertions then select") {
+    val tableName = Person.generatePersonTableName()
+    // TODO: retrieve table name from tableName instead of path option
+    spark.sql(
+      s"""CREATE TEMPORARY TABLE $tableName (name STRING, age INT, address STRING, salary DOUBLE)
+         |  USING $RedisFormat OPTIONS (path '$tableName')
+         |""".stripMargin)
+    spark.sql(
+      s"""INSERT INTO TABLE $tableName
+         |  VALUES ('John', 30, '60 Wall Street', 150.5),
+         |    ('Peter', 35, '110 Wall Street', 200.3)
+         |""".stripMargin)
+    val loadedDf = spark.sql(
+      s"""SELECT * FROM $tableName
```

Comment:
Please reformat to a single line  `val loadedDf = spark.sql(s"SELECT * FROM $tableName")`. The same applies to methods below.